### ISSUE TITLE
Launcher should provide the option to uninstall context menu integration

### ISSUE THREAD

**Issue by torusrxxx** [x64dbg/x64dbg#3138]

**Comment by foralost** [comment]
`context menu integration`

what is that?

**Comment by torusrxxx** [comment]
Yes. To uninstall `Debug with x64dbg` entries, the user have to come to https://github.com/x64dbg/x64dbg/wiki/Frequently-Asked-Questions and download a [regfile](https://raw.githubusercontent.com/x64dbg/x64dbg/development/bin/x64dbg_shell_remove.reg). This should be in the launcher instead, the user shouldn't reach out to web to uninstall it.

**Comment by mrexodia** [comment]
I honestly don’t think it’s very important to improve this. It would require some annoying changes to the launcher UI, which is currently very simple and that’s probably a benefit. I have similar reservations about #3316. It’s trivial to restart x64dbg as admin already

**Comment by torusrxxx** [comment]
Maybe rename the third button in launcher from "install" to "configuration"? Not providing an uninstaller that uninstalls everything puts the software into the same category as many spyware/adware that you must download another removal tool to remove it.

**Comment by torusrxxx** [comment]
Also on #3316, it's easy to restart x64dbg as admin but I'll still prefer a desktop shortcut when I just want to run it elevated every time. 10 seconds of launch/relaunch time and additional menu clicks vs 1 second, every time x64dbg is launched. Plus there's not a easy way to create the admin shortcut.

**Comment by mrexodia** [comment]
Yeah I think renaming it to “configuration” is fine. As for the admin shortcut you can do it yourself very easily…

**Comment by G3ph4z** [comment]
I've started working on this along with #1640, I think it goes together.

**Comment by G3ph4z** [comment]
PR got merged, so this can be closed.
